<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mediaEditor</class>
 <widget class="QWidget" name="mediaEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>570</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>570</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>570</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create new media</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="yearLabel">
     <property name="text">
      <string>Year</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="durationLine"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="photoLabel">
     <property name="text">
      <string>Image</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="authorLine"/>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="photoButton">
     <property name="text">
      <string>Add image</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="durationLabel">
     <property name="text">
      <string>Duration</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="titleLine"/>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="yearLine"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="authorLabel">
     <property name="text">
      <string>Author</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="audiobook">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="readerLabel">
         <property name="text">
          <string>Reader</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="readerLine"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="summaryLabel">
         <property name="text">
          <string>Summary</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QWidget" name="widget" native="true">
         <widget class="QTextEdit" name="summaryText">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>311</width>
            <height>261</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="music">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="albumLabel">
         <property name="text">
          <string>Album</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="albumLine"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lyricsLabel">
         <property name="text">
          <string>Lyrics</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QWidget" name="widget_2" native="true">
         <widget class="QTextEdit" name="lyricsText">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>331</width>
            <height>271</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="podcast">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QLabel" name="episodeLabel">
         <property name="text">
          <string>Episode</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="episodeLine"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="sesaonLabel">
         <property name="text">
          <string>Season</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="seasonLine"/>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
